src/modules/common/DroppableLabel.tsx

```diff
@@ -41,39 +43,48 @@ type Props = {
   isDragging?: boolean;
   isSubmitted: boolean;
   index: number;
+  onRemoveLabel: DraggableLabelToDroppableContProps['onRemoveLabel'];
 };
 
 const DroppableLabel = ({
   label,
   isDragging,
   isSubmitted,
   index,
-}: Props): JSX.Element => (
-  <Droppable droppableId={label.expected.id} key={label.expected.id}>
-    {(provided, dropSnapshot) => (
-      <Wrapper
-        ref={provided.innerRef}
-        {...provided.droppableProps}
-        top={label.expected.y}
-        left={label.expected.x}
-        isFilled={Boolean(label.actual)}
-        isDraggingOver={dropSnapshot.isDraggingOver}
-        isDragging={isDragging}
-        id={buildDraggableLabelId(label.expected.id)}
-      >
-        {label.actual ? (
-          <DraggableLabelToDroppableCont
-            isCorrect={label.expected.id === label.actual?.id}
-            isSubmitted={isSubmitted}
-            content={label.actual.content}
-            index={index}
-            draggableId={label.actual.id}
-          />
-        ) : null}
-        {provided.placeholder}
-      </Wrapper>
-    )}
-  </Droppable>
-);
+  onRemoveLabel,
+}: Props): JSX.Element => {
+  // instance exists if within transform wrapper: inside frame
+  const { instance } = useControls();
+
+  return (
+    <Droppable droppableId={label.expected.id} key={label.expected.id}>
+      {(provided, dropSnapshot) => (
+        <Wrapper
+          ref={provided.innerRef}
+          {...provided.droppableProps}
+          top={label.expected.y}
+          left={label.expected.x}
+          isFilled={Boolean(label.actual)}
+          isDraggingOver={dropSnapshot.isDraggingOver}
+          isDragging={isDragging}
+          id={buildDraggableLabelId(label.expected.id)}
+        >
+          {label.actual ? (
+            <DraggableLabelToDroppableCont
+              isCorrect={label.expected.id === label.actual?.id}
+              isSubmitted={isSubmitted}
+              content={label.actual.content}
+              index={index}
+              label={label.actual}
+              isDragDisabled={Boolean(instance)}
+              onRemoveLabel={onRemoveLabel}
+            />
+          ) : null}
```

Comment:
Not sure if the `Boolean` is required, but what about: 
```suggestion
          {Boolean(label.actual) && (
            <DraggableLabelToDroppableCont
              isCorrect={label.expected.id === label.actual?.id}
              isSubmitted={isSubmitted}
              content={label.actual.content}
              index={index}
              label={label.actual}
              isDragDisabled={Boolean(instance)}
              onRemoveLabel={onRemoveLabel}
            />
          )}
```